### PR TITLE
[4.2] Support VPCs when creating droplets

### LINKED ISSUE #268: Create droplet private_networking property deprecated
In reference to [the docs](https://developers.digitalocean.com/documentation/v2/#create-a-new-droplet), the `private_networking` attribute has been deprecated as of April 2020 and should be replaced with the new `vpc_uuid` attribute.

### PR DESCRIPTION
**Pull request by LewisSmallwood**
Fixes #268

### CODE REVIEW

**Comment by LewisSmallwood** [comment]
Hi @GrahamCampbell, I've closed the other PR in favour of this one without the shared commits.

File: src/Api/Droplet.php [hunk 1]
```diff
@@ -93,18 +93,19 @@ public function getById(int $id)
      * @param string|int   $image
      * @param bool         $backups
      * @param bool         $ipv6
-     * @param bool         $privateNetworking
+     * @param bool         $privateNetworking Deprecated, use vpcUuid
```
Comment (GrahamCampbell) [inline src/Api/Droplet.php]: Better to rename this parameter and have it accept `string|bool`.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Api/Droplet.php (hunk 1 above)
Comment (LewisSmallwood) [reply src/Api/Droplet.php]: No problem, I'll make this change. To confirm it should be `bool|string $privateNetworking` or `bool|string $vpcUuid`?

**Comment by GrahamCampbell** [comment]
Thanks. :)

**Pull request merged**